components/webcompat_reporter/common/webcompat_reporter.mojom

```diff
@@ -30,4 +30,7 @@ struct ReportInfo {
 
 interface WebcompatReporterHandler {
   SubmitWebcompatReport(ReportInfo report_info);
+  GetContactInfoSaveFlag() => (bool val);
+  SetContactInfoSaveFlag(bool value);
+  GetContactInfo() => (string? val);
```

Comment:
do we need the "getter" calls? could we not just include the information when opening the dialog via `WebcompatReporterDialogDelegate::OpenReporterDialog` ?